contracts/metatx/ERC2771Context.sol

```diff
@@ -18,27 +18,54 @@ abstract contract ERC2771Context is Context {
     /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
     address private immutable _trustedForwarder;
 
+    /**
+     * @dev Sets an immutable trusted forwarder.
```

Comment:
```suggestion
     * @dev Initializes the contract with a trusted forwarder, which will be able to invoke functions on
     * this contract on behalf of other accounts.
```

Reply:
Fixed in [8309315](vscode-file://vscode-app/Applications/Visual%20Studio%20Code.app/Contents/Resources/app/out/vs/code/electron-sandbox/workbench/workbench.html)